saviSimulation/savi/jason_processing/UxV.java

```diff
@@ -69,14 +69,14 @@ public void draw(PVector position) {
 		image=simulator.loadShape("SimImages/"+imageName+".svg");
 		
 		// translate to center image on uasposition.x, uasposition.y
-		//	simulator.shapeMode(PConstants.CENTER); didn't work
+//			simulator.shapeMode(PConstants.CENTER);// didn't work
 		image.translate(-image.width/2,-image.height/2);		
 		
 		// to adjust compassAngle to the image
 		image.rotate((float) ((float)this.getBehavior().getCompassAngle()+Math.PI/2));
 
 		//draw image
```

Comment:
could you explain what these changes are for (drawing the shape in UxV instead of WorldObject, and having this position.x +pixels/2 )?
I'm not objecting to it, I just would like to understand

Reply:
I just rewrote the data loading from file using fancy streams, I hope that's ok; behaviour is unchanged.
I think this is ready to merge.

Reply:
@ntartania I am not sure to which part of the code do you refer. I guess you talk about the one that rotates the image so the UxV face the direction it is moving? The other world object does not need this rotation